vault/resource_pki_secret_backend_config_cluster.go

```diff
@@ -0,0 +1,142 @@
+// Copyright (c) HashiCorp, Inc.
+// SPDX-License-Identifier: MPL-2.0
+
+package vault
+
+import (
+	"context"
+	"fmt"
+	"log"
+	"strings"
+
+	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/schema"
+
+	"github.com/hashicorp/terraform-provider-vault/internal/consts"
+	"github.com/hashicorp/terraform-provider-vault/internal/provider"
+	"github.com/hashicorp/terraform-provider-vault/util"
+)
+
+var pkiClusterFields = []string{
+	consts.FieldPath,
+	consts.FieldAIAPath,
+}
+
+func pkiSecretBackendConfigClusterResource() *schema.Resource {
+	return &schema.Resource{
+		Create: pkiSecretBackendConfigClusterCreateUpdate,
```

Comment:
I believe this feature was added in Vault 1.13 so we should probably add a version check similar to https://github.com/hashicorp/terraform-provider-vault/blob/52650fe4752ff6e2c2f3ee3db7b1478057c413dd/vault/resource_pki_secret_backend_issuer.go#L26

Reply:
I added the check for version 1.13